Calibration/PPSAlCaRecoProducer/python/PPSAlCaRecoProducer_cff.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+import copy
+from HLTrigger.HLTfilters.hltHighLevel_cfi import *
```

Comment:
Maybe if you import the whole module instead of each individual parameter, you won't need to use the deepcopy.
You could just do 
import HLTrigger.HLTfilters.hltHighLevel_cfi

Reply:
Good point, I will check it.

Reply:
It seems this works, I've added a change in 3303867 (following the style used in https://github.com/cms-sw/cmssw/blob/master/HLTrigger/HLTfilters/test/hltHighLevel.py)
is it in line with what you suggested ?

Reply:
Partially yes, but the comment in https://github.com/cms-sw/cmssw/pull/36273#pullrequestreview-820388994 was actually more important, please move to using the trigger bits tag in the GT instead of hardcoded triggers, thanks!

Reply:
OK, I will investigate it. First I need to get familiar with this mechanism. Do you mean modifying GT and then reading it via `AlCaRecoTriggerBitsRcd` from CondDB by setting proper GT ?

I am guessing that from the logic of `HLTHighLevel` EDFilter:
- https://github.com/cms-sw/cmssw/blob/master/HLTrigger/HLTfilters/plugins/HLTHighLevel.cc#L73
- https://github.com/cms-sw/cmssw/blob/master/HLTrigger/HLTfilters/plugins/HLTHighLevel.cc#L96

Is there any documentation for this part somewhere ?

Reply:
Here is a commit which moved from hardcoded path to a trigger bit one:
https://github.com/cms-sw/cmssw/commit/721ef2399b18483f2b59873f30f75e7217b31cc5#diff-0449e48c7dccf73ad9619453833f658f86cec5a32468a71ff0a32aede308f0e9

This is the change needed from your side, the rest is an update of the `AlCaRecoTriggerBitsRcd` which is on our side ( (attn. @hjkwon260 )

Reply:
@grzanka since this is done now, please resolve this comment